test/Microsoft.AspNet.Mvc.FunctionalTests/XmlSerializerInputFormatterTests.cs

```diff
@@ -46,9 +46,9 @@ public class XmlSerializerInputFormatterTests
             var sampleInputInt = 10;
             var input = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" +
                 "<DummyClas xmlns:i=\"http://www.w3.org/2001/XMLSchema-instance\" " +
-                "i:type=\"DerivedDummyClass\" xmlns=\"http://schemas.datacontract.org/2004/07/XmlSerializerWebSite\">" +
+                "i:type=\"DerivedDummyClass\" xmlns=\"http://schemas.datacontract.org/2004/07/XmlFormattersWebSite\">" +
                 "<SampleInt>" + sampleInputInt.ToString() + "</SampleInt></DummyClass>";
-            var content = new StringContent(input, Encoding.UTF8, "application/xml");
+            var content = new StringContent(input, Encoding.UTF8, "application/xml-xmlser");
```

Comment:
The XmlFormattersWebSite(which was originally called XmlSerializerWebSite) has both the DCS and XmlSerializer based formatters and since both of these formatters have supported media types like `application/xml` and `text/xml`, I needed a way to distinguish in invoking a particular formatter for testing...so we changed these supported media types in the website's Startup.cs file...this way tests can use the same website for DCS or XmlSerializer formatter based testing...